### ISSUE TITLE
Emulators fail request to cloud-firestore-emulator-v1.11.11.jar

### ISSUE THREAD

**Issue by PawFV** [firebase/firebase-tools#2953]
### Environment info

**firebase-tools:** 9.0.1 

**Platform:** Windows 10, node v12

### Description

Runing emulators ui fails attempting to make a request to: **https://storage.googleapis.com/firebase-preview-drop/emulator/cloud-firestore-emulator-v1.11.11.jar**

#### Logs:
```console
i  emulators: Starting emulators: auth, functions, firestore, pubsub
[0] !  functions: You are running the functions emulator in debug mode (port=9229). This means that functions will execute in sequence rather than in parallel.
[0] !  functions: The following emulators are not running, calls to these services from the Functions emulator will affect production: database, hosting       
[0] +  functions: Using node@12 from host.
[1] 
[1] 16:12:37 - Found 0 errors. Watching for file changes.
[0] i  firestore: Importing data from C:\Users\prego\OneDrive\Escritorio\BIMMEET\server\emulatorData\firestore_export\firestore_export.overall_export_metadata
[0] i  firestore: downloading cloud-firestore-emulator-v1.11.11.jar...
[0] i  emulators: Shutting down emulators.   
[0] i  functions: Stopping Functions Emulator
[0] i  firestore: Stopping Firestore Emulator
[0] i  hub: Stopping emulator hub
[0]
[0] Error: Failed to make request to https://storage.googleapis.com/firebase-preview-drop/emulator/cloud-firestore-emulator-v1.11.11.jar
[0] npm ERR! code ELIFECYCLE
[0] npm ERR! errno 1
[0] npm ERR! functions@ start:emulators: `firebase emulators:start --inspect-functions --import=../emulatorData`
```

### Expected behavior
Running firebase locally after updating to 9.0.1 from 8.2

### Actual behavior

Fails at downloading cloud-firestore-emulator-v1.11.11.jar

**Comment by samtstern** [comment]
@PawFV I'm not familiar with what `ELIFECYCLE` means but this StackOverflow post suggests you may want to try running the command as an Administrator. It may be a permissions error with downloading the Emulator JAR.
https://stackoverflow.com/questions/30744964/what-does-the-elifecycle-node-js-error-mean/40377737

Also can you confirm that you're able to access this link from your web browser?
https://storage.googleapis.com/firebase-preview-drop/emulator/cloud-firestore-emulator-v1.11.11.jar

**Comment by PawFV** [comment]
Hello, for some reason when I opened it before making this post I got an error, it said the connection was not secure, but I wasn't sure to post it here.

The next day I tried it again and everything worked just fine.

Thanks for the support, it is fixed now.

**Comment by samtstern** [comment]
Thanks for following up!

**Comment by mancku** [comment]
Hi @samtstern,  

I just had the same issue, and can confirm that after getting the error a couple of times,  I found that issue, followed your instructions, tried to open https://storage.googleapis.com/firebase-preview-drop/emulator/cloud-firestore-emulator-v1.11.11.jar from the browser, and I got an alert because apparently my connection was insecure. Closed the browser tab, opened it again and that time it worked from the browser. I'm talking of a 10 seconds span from the first attempt to the second.
After that, I went to the command line, tried it again and then it worked.

So, I'm guessing some connections get blocked for a reason I can't understand.
Hope it helps find the issue.

Node version:  12.18.1
Firebase-Tools version: 9.1.0
Browser: Microsoft edge 87.0.664.66 
SO: Microsoft Windows [Version 10.0.18363.1256]
